pkg/apis/cyclone/v1alpha1/project.go

```diff
@@ -0,0 +1,47 @@
+package v1alpha1
+
+import (
+	core_v1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Project defines a project which holds a lot of common default information of some workflow under it.
```

Comment:
// Project defines a project which holds common information of workflows under it.